cape/cape_result.py

```diff
@@ -2243,7 +2293,14 @@ def process_buffers(
                     ):
                         buffer_body.append(table_row)
                         count_per_source_per_process += 1
-
+                        network_buffers.append(table_row)
+    if len(network_buffers) > 0:
```

Comment:
We do not want to extract all buffers. We are only interested in extracting buffers that contain the PE_INDICATORS. Also we do not want to write all buffers that contain the PE_INDICATORS to a single file. We want one file per buffer that contains PE_INDICATORS, which is why I mentioned using a prefix instead of a file name.